HW_1/task_2.py

```diff
@@ -0,0 +1,39 @@
+# Была выполнена 2 задача из домашнего задания номер 1.
+
+""" 
+Напишите код, который запрашивает число и сообщает является ли оно простым или составным. 
+Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”. 
+Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч. 
+"""
+
+MIN_LIMIT = 0
+MAX_LIMIT = 100000
+count = 3
+i = 1
+while i <= count:
+	print('Попытка ' + str(i))
+	i += 1
+	num = int(input('Введите число между ' + str(MIN_LIMIT) + ' и ' +	str(MAX_LIMIT) + ': '))
+	if num < MIN_LIMIT or num > MAX_LIMIT:
+		print('Введенное число вне ограничений от 0 до 100000, введите корректное число: ')
+	else:
+		break
+else:
+	print('Вы ввели некорректные данные трижды. Программа завершена.')
+	quit()
+
+print('Было введено число ' + str(num) + ', проверка на простое/составное: ')
+
+if num < 4:
+	print('Число простое')
+	quit()
+
+divider = 2
+while divider <= round(num**0.5):
+	if (num % divider) == 0:
+		print('Это число составное')
+		break
+	divider += 1
+else:
+	print('Это число простое')
```

Comment:
Не забывайте приводить к PEP8
```suggestion
    print('Попытка ' + str(i))
    i += 1
    num = int(input('Введите число между ' + str(MIN_LIMIT) + ' и ' + str(
        MAX_LIMIT) + ': '))
    if num < MIN_LIMIT or num > MAX_LIMIT:
        print(
            'Введенное число вне ограничений от 0 до 100000, введите корректное число: ')
    else:
        break
else:
    print('Вы ввели некорректные данные трижды. Программа завершена.')
    quit()

print('Было введено число ' + str(num) + ', проверка на простое/составное: ')

if num < 4:
    print('Число простое')
    quit()

divider = 2
while divider <= round(num ** 0.5):
    if (num % divider) == 0:
        print('Это число составное')
        break
    divider += 1
else:
    print('Это число простое')
```

Reply:
у меня в VSC все нормально выглядит - это GitHub почему-то так растягивает. Не могу понять почему так.